src/components/helpers/ajax.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios'
+
+class Ajax {
+  fetchTags () {
+    return axios.get('http://localhost:3001/api/v1/tags.json')
```

Comment:
I'm unfamiliar with axios, but it seems like this hard-coded URL may not work in production.